--- /Users/mneuhaus/Sites/FLOW3-Vanilla/Packages/Application/TYPO3.TYPO3/Classes/Routing/BackendModuleRoutePartHandler.php 
+++ (clipboard) 
@@ -68,7 +68,8 @@
 					$moduleController = isset($moduleConfiguration['controller']) ? $moduleConfiguration['controller'] : '\TYPO3\TYPO3\Controller\Module\StandardController';
 				} else {
 					if ($level === count($segments)) {
-						if ($this->reflectionService->hasMethod($moduleController, $segment . 'Action') === TRUE) {
+						if ($this->reflectionService->hasMethod($moduleController, $segment . 'Action') === TRUE
+							 || $this->reflectionService->hasMethod($moduleController, 'dispatch') === TRUE) {
 							$moduleAction = $segment;
 							break;
 						}
