--- Application/TYPO3.Form/Classes/Core/Runtime/FormRuntime.php 
+++ (clipboard) 
@@ -126,11 +126,15 @@
 	 */
 	public function __construct(\TYPO3\Form\Core\Model\FormDefinition $formDefinition, \TYPO3\FLOW3\Mvc\ActionRequest $request, \TYPO3\FLOW3\Http\Response $response) {
 		$this->formDefinition = $formDefinition;
+		$rootRequest = $request->getMainRequest() ?: $request;
+		$pluginArguments = $rootRequest->getPluginArguments();
 		$this->request = new ActionRequest($request);
-		$this->request->setArgumentNamespace($this->formDefinition->getIdentifier());
-		if ($this->request->getParentRequest()->hasArgument($this->request->getArgumentNamespace()) === TRUE && is_array($this->request->getParentRequest()->getArgument($this->request->getArgumentNamespace()))) {
-			$this->request->setArguments($this->request->getParentRequest()->getArgument($this->request->getArgumentNamespace()));
-		}
+		$formIdentifier = $this->formDefinition->getIdentifier();
+		$this->request->setArgumentNamespace('--' . $formIdentifier);
+		if (isset($pluginArguments[$formIdentifier])) {
+			$this->request->setArguments($pluginArguments[$formIdentifier]);
+		}
+
 		$this->response = $response;
 	}
 
