--- /Users/mneuhaus/Sites/FLOW3/Phoenix/Packages/Framework/TYPO3.FLOW3/Classes/Property/PropertyMappingConfiguration.php 
+++ (clipboard) 
@@ -70,6 +70,13 @@
 	protected $mapUnknownProperties = FALSE;
 
 	/**
+	 * If TRUE, unknown properties will be mapped. recursivly
+	 *
+	 * @var boolean
+	 */
+	protected $mapUnknownPropertiesRecursivly = FALSE;
+
+	/**
 	 * The behavior is as follows:
 	 *
 	 * - if a property has been explicitely forbidden using allowAllPropertiesExcept(...), it is directly rejected
@@ -104,6 +111,18 @@
 	}
 
 	/**
+	 * Allow all properties in property mapping, even unknown ones.
+	 *
+	 * @return \TYPO3\FLOW3\Property\PropertyMappingConfiguration this
+	 * @api
+	 */
+	public function allowAllPropertiesRecursivly() {
+		$this->allowAllProperties();
+		$this->mapUnknownPropertiesRecursivly = TRUE;
+		return $this;
+	}
+
+	/**
 	 * Allow a list of specific properties. All arguments of
 	 * allowProperties are used here (varargs).
 	 *
@@ -147,6 +166,12 @@
 	public function getConfigurationFor($propertyName) {
 		if (isset($this->subConfigurationForProperty[$propertyName])) {
 			return $this->subConfigurationForProperty[$propertyName];
+		}
+
+		if($this->mapUnknownPropertiesRecursivly){
+			$subConfigurationForProperty = new \TYPO3\FLOW3\Property\PropertyMappingConfiguration();
+			$subConfigurationForProperty->allowAllPropertiesRecursivly();
+			return $subConfigurationForProperty;
 		}
 
 		return new \TYPO3\FLOW3\Property\PropertyMappingConfiguration();
