--- Framework/TYPO3.FLOW3/Classes/Property/PropertyMapper.php 
+++ (clipboard) 
@@ -155,9 +155,14 @@
 	 * @throws \TYPO3\FLOW3\Property\Exception\InvalidPropertyMappingConfigurationException
 	 */
 	protected function doMapping($source, $targetType, \TYPO3\FLOW3\Property\PropertyMappingConfigurationInterface $configuration, &$currentPropertyPath) {
+		if (is_object($source) && ($source instanceof $targetType)) {
+			return $source;
+		}
+
 		if ($source === NULL) {
 			$source = '';
 		}
 
 		$typeConverter = $this->findTypeConverter($source, $targetType, $configuration);
 
